### PR TITLE
Fix typos across repo

### PR DESCRIPTION
**Pull request by stavares843**
This PR adds the following:
- Fix typos across repo

### CODE REVIEW

**Comment by david-ros** [comment]
@stavares843  I think the word "add-ons" was correctly spelled (vs proposed "add-owns"), could you modify so the other changes are preserved without changing to add-owns please?

**Comment by stavares843** [comment]
will do, i also thought was in purpose but then i noticed the file name still says add-ons :)

File: docs/kagi/support-and-community/contribute.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ The Kagi user community is a vibrant, diverse, and passionate group that plays a
 - **Contribute to Kagi documentation**: Help us in enhancing our documentation. Each page on our site is editable; feel free to add or amend content using the link at the bottom of each page. Contributions are typically merged on the same day.
 - **Translation**: Assist in [translating Kagi](./contribute_translations.md) into other languages, helping us reach a wider audience globally.
 - **Work on Kagi open-source projects**: If you are interested in open-source projects, we have a [range of projects](./open-source.md) you can contribute to.
-- **Develop add-ons**: Use your creativity and skills to develop [add-ons for Kagi](../community-addons/index.md), enhancing its functionality and user experience.
+- **Develop add-owns**: Use your creativity and skills to develop [add-owns for Kagi](../community-addons/index.md), enhancing its functionality and user experience.
```
Comment (stavares843) [inline docs/kagi/support-and-community/contribute.md:11]:
```suggestion
- **Develop add-ons**: Use your creativity and skills to develop [add-ons for Kagi](../community-addons/index.md), enhancing its functionality and user experience.
```

**Comment by stavares843** [comment]
ready @david-ros

**Pull request merged**